File: app/src/main/java/com/example/notes/title.java
package com.example.notes;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
public class title extends AppCompatActivity {


    private static int splash_time_out = 1000;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_title);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent main = new Intent(title.this, StartActivity.class);
                startActivity(main);
                finish();
            }
        },splash_time_out);

    }


}

File: app/src/main/java/com/example/notes/title.kt
package com.example.notes

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class title : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_title)
        Handler().postDelayed({
            val main = Intent(this@title, StartActivity::class.java)
            startActivity(main)
            finish()
        }, title.Companion.splash_time_out.toLong())
    }

    companion object {
        private const val splash_time_out = 1000
    }
}